.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor
 composer.lock
 phpunit.xml
+.idea/
```

Comment:
Isn't it better to this one into global gitignore as it's related to specific dev env?